index.Rmd

```diff
@@ -515,8 +527,33 @@ plot(forest, col = "dark green", legend = FALSE)
 
 # Today's summary
 
-We learned about:
+Today you performed a supervised classification, you identified patches and sieve connected cells, and you learned to deal with thematic raster data. Some functions to retain: 
+
+## Data exploration 
+
+* `hist()`: Create a histogram for each layer of a `SpatRaster`.
+* `pairs()`: Create a scatterplot for each pair of layers of a `SpatRaster`. 
+* `app()`: Apply a function to all pixels of a `SpatRaster` more efficiently. 
+
+## Training data preparation 
+
+* `extract()`: Retrieve a value for the raster below a polygon. 
+
+## Contruct a Random Forest model
+
+* `ranger()`: Build a Random Forest model based on a data frame of predictors and a response variable. `importance = "permutation"` and `importance()` to get the statistical importance of each predictor. 
+
+## Run a model on the data
+
+* `predict()`: Predict raster values based on a predefined model. 
+
+## Applying a raster sieve by identifying patches 
+
+* `setValues()`: Assign a new value to a raster
```

Comment:
This is not entirely clear; rather, `setValues()` sets all values of a raster to a certain value or certain values. This is equivalent to `MySpatRaster[] = MyValue`.